airbyte-webapp/src/components/Modal/ModalFooter.module.scss

```diff
@@ -0,0 +1,18 @@
+@use "../../scss/colors";
+@use "../../scss/variables";
+
+.modalFooter {
+  display: flex;
+  flex-direction: row;
+  justify-content: flex-end;
+  border-top: variables.$border-thin solid colors.$grey-100;
+  padding: variables.$spacing-lg variables.$spacing-xl;
+
+  & > * {
+    margin-left: variables.$spacing-md;
+
+    &:first-child {
+      margin-left: 0;
+    }
+  }
```

Comment:
This should most likely just be a `gap: variables.$spacing-md`

Reply:
I didn't know gap was a property that I could use! Will update...

Reply:
Luckily the pain was so large with those margin solutions (and doesn't work when multi line), that browser vendors were extremly quickly in adding those also for flex, after it was added for the grid layout :D https://caniuse.com/flexbox-gap